src/Swift.Runtime/src/Swift/AnyType.cs

```diff
@@ -46,10 +47,23 @@ IntPtr ISwiftObject.MarshalToSwift(IntPtr swiftDest)
         {
             throw new InvalidOperationException("Cannot marshal AnyType to Swift without payload");
         }
-        unsafe {
-            metadata.ValueWitnessTable->InitializeWithCopy((void *)swiftDest, (void *)_payload, metadata);
+        unsafe
+        {
+            metadata.ValueWitnessTable->InitializeWithCopy((void*)swiftDest, (void*)_payload, metadata);
         }
         return swiftDest;
     }
 
+    /// <summary>
+    /// Gets the protocol conformance descriptor for the given type
+    /// </summary>
+    /// <typeparam name="TProtocol"></typeparam>
+    /// <returns></returns>
+    static ProtocolConformanceDescriptor ISwiftObject.GetProtocolConformanceDescriptor<TProtocol>()
+        where TProtocol : class
+    {
+        // TODO: Implement
```

Comment:
```suggestion
        // TODO: https://github.com/dotnet/runtimelab/issues/2957
```

Please update the tracking issue if needed.